### PR TITLE
CO-1569 Les attendance day ne sont plus créer chaque jour automatique…

### PR DESCRIPTION
**Pull request by vwvw**
…ment

The cron job was crashing due to a wrong column name in the SQL query.-mAdded a check to create attendance_day only for employee which currently have a valid contract.-mExchanged two columns and moved the number from right aligned to left aligned with the help of type="char"

### CODE REVIEW

**Comment by eicher31** [comment]
Some PEP8 Error:
======== Testing test_flake8 ========
hr_attendance_calendar/models/hr_attendance_day.py:208:39: E126 continuation line over-indented for hanging indent
hr_attendance_calendar/models/hr_attendance_day.py:209:35: E127 continuation line over-indented for visual indent
hr_attendance_calendar/models/hr_attendance_day.py:335:13: E125 continuation line with same indent as next logical line

File: hr_attendance_calendar/models/hr_attendance_day.py [hunk 1]
```diff
@@ -125,8 +125,11 @@ def _compute_due_hours(self):
                 continue
 
             # Contract
+            # sum the due hours from today attendance prescription for the
+            # employee. These attendance are selected in
+            # update_calendar_attendance() which is called upon
+            # attendance_day creation.
```
Comment (eicher31) [inline hr_attendance_calendar/models/hr_attendance_day.py]: I didn't really understand your comment... where is the method update_calendar_attendance?

Comment (vwvw) [reply hr_attendance_calendar/models/hr_attendance_day.py]: Further down in the file

**Review by eicher31 — APPROVED** [review]
It's OK for me! Just fix the pep8 errors

**Review by ecino — APPROVED** [review]

**Comment by ecino** [comment]
hr_attendance_calendar/models/hr_attendance_day.py:336:25: E126 continuation line over-indented for hanging indent

**Pull request merged**